lib/facility_factory.rb

```diff
@@ -0,0 +1,37 @@
+class FacilityFactory
+
+  def initialize
```

Comment:
You don't need this method. We aren't instantiating a new `FacilityFactory` with any attributes/args.